core/db/migrate/20160101010000_solidus_one_four.rb

```diff
@@ -426,10 +426,12 @@ def up
       t.integer "property_id"
       t.datetime "created_at", precision: 6
       t.datetime "updated_at", precision: 6
+      t.datetime "deleted_at", precision: 6
```

Comment:
I think we can't change this migration but we need a new one, otherwise who updates Solidus won't take this database change.

Reply:
I added it here because of this error https://circleci.com/gh/solidusio/solidus/12317, do you know if I need to do something else to detect the new migration?